pennylane_lightning/src/gates/cpu_kernels/GateImplementationsPI.hpp

```diff
@@ -922,6 +923,30 @@ class GateImplementationsPI : public PauliGenerator<GateImplementationsPI> {
         return -static_cast<PrecisionT>(0.5);
     }
 
+    template <class PrecisionT>
+    [[nodiscard]] static auto
+    applyGeneratorIsingXY(std::complex<PrecisionT> *arr, size_t num_qubits,
+                          const std::vector<size_t> &wires,
+                          [[maybe_unused]] bool adj) -> PrecisionT {
+        PL_ASSERT(wires.size() == 2);
+        const auto [indices, externalIndices] = GateIndices(wires, num_qubits);
+
+        for (const size_t &externalIndex : externalIndices) {
+            std::complex<PrecisionT> *shiftedState = arr + externalIndex;
+            /*
+            const auto v01 = shiftedState[indices[1]];
+            shiftedState[indices[1]] = shiftedState[indices[2]];
+            shiftedState[indices[2]] = v01;
+            */
```

Comment:
```suggestion
```

Reply:
Do you mean I need to remove it? Thanks

Reply:
Yes. When with a red background, and without a text with a green background, it means a suggestion to remove the text. You can click Commit suggestion, or implement the change locally and commit and push the changes later.

Reply:
Thanks @AmintorDusko  for the clarification!